app/models/master_file.rb

```diff
@@ -723,6 +723,18 @@ def post_processing_file_management
     case Settings.master_file_management.strategy
     when 'delete'
       MasterFileManagementJobs::Delete.perform_now self.id
+    when 'move-ui-upload-only'
+      # check if the MasterFile.file_location contains the `dropbox_directory_name`
+      # If video uploaded via the UI file upload then video stored in temporary upload directory
+      # Move to a directory named after the collection
+      dropbox_directory_name = self.media_object.collection.dropbox_directory_name
+      raise '"dropbox_directory_name" missing for master_file_management strategy "move-ui-upload-only"' if dropbox_directory_name.blank?
+      if self.file_location.exclude? dropbox_directory_name
+        move_path = Settings.master_file_management.path
+        raise '"path" configuration missing for master_file_management strategy "move"' if move_path.blank?
```

Comment:
I know this is to "move" files from temp to `dropbox_directory_name`, but this error message says 'for master_file_management strategy "move"' - could be misleading as this is within the strategy `move_ui_upload_only`? Maybe `raise '"path" configuration missing for master_file_management strategy "move_ui_upload_only"' if move_path.blank?`

Reply:
Good suggestion!